collector/mdadm_linux.go

```diff
@@ -98,6 +104,31 @@ var (
 		[]string{"device"},
 		nil,
 	)
+
+	blocksToBeSyncedDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "md", "blocks_tobesynced"),
+		"Number of blocks on the device that need to be synced.",
+		[]string{"device"},
+		nil,
+	)
+	blocksSyncedPctDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "md", "blocks_synced_pct"),
```

Comment:
Ideally we would have a blocks_synced so this can be calculated from blocks_to_by_synced

Reply:
Ah I missed this. In that case just remove the blocks_synced_pct. This can be easily and cheaply calculated in PromQL and therefor is best practice to not expose it